woocommerce/woocommerce
# Adapt the width of the pricing fields

### All Submissions:






### Changes proposed in this Pull Request:

Changes the width of the pricing fields based on the screen's width.

I'm not sure how we want to determine the width of the fields. @jarekmorawski mentioned the 1032px width rule on issue #35180, so that's what I adopted in the first place.



Closes #35538.






### How to test the changes in this Pull Request:



1. Go to the new Add Product page (/wp-admin/admin.php?page=wc-admin&path=%2Fadd-product)
2. Check if the width of the pricing fields are 50% when the screen width is larger than 1032px
3. Adjust the screen width to a smaller value and check if the width of the pricing fields turn into 100%



### Other information:




### FOR PR REVIEWER ONLY:
